data/brands/amenity/cafe.json

```diff
@@ -3663,6 +3663,19 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Thelins konditori",
+      "id": "thelinskonditori-46f791",
+      "locationSet": {"include": ["se"]},
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Thelins konditori",
+        "brand:wikidata": "Q124048792",
```

Comment:
It only has 18 branches shop. Does it really satisfy the notability?

Reply:
Yeah that's a good question. I've seen some other smaller chains in the index before, but I'm also fine with not adding it :)

Reply:
I think 18 is ok as a number, but the first thing I'm going to look at is Q124048792 and right now it looks pretty empty or spammy.  I worry that Wikidata admins might remove it if you don't add some supporting details from other sources.

Reply:
that's all the data I have for them right now. What do you think would be missing?